client/src/pages/SignUp.jsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Style from '../css/Login.module.css';
+import axios from 'axios';
+
+const SignUp = () => {
+  const [apelido, setApelido] = useState('');
+  const [isSenhaActive, setSenhaActive] = useState(false);
+  const [senha, setSenha] = useState('');
+  const [segundaSenha, setSegundaSenha] = useState('');
+
+  const CreateUser = () => {
+    const newUser = { apelido, senha };
```

Comment:
Podemos padronizar as rotas de SignIn (Login) para /sign-in e SignUp para /sign-up, o que acha?

Reply:
po nice dms, assino embaixo!